public/_templates/bulma/css/forum.css

```diff
@@ -116,3 +116,7 @@ a.neutral-link {
 .navbar-end form {
   width: 100%;
 }
+@media screen and (min-width: 1024px)
```

Comment:
Missing bracket:

```suggestion
@media screen and (min-width: 1024px) {
```

Reply:
Good catch :) I think a second closing one is missing, too.